docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +6,253 @@ The parent key for all of the following parameters is `lieutenant`.
+
+
+== `operator.vault.path`
+
+[horizontal]
+type:: string
+default:: `kv`
+
+The mount path of the KV secret engine to be used.
+
+
+== `api.version`
+
+[horizontal]
+type:: string
+default:: `v0.6.0`
+
+The specific lieutenant-api version to deploy.
+
+
+
+== `api.image`
+
+[horizontal]
+type:: string
+default:: `docker.io/projectsyn/lieutenant-api:v0.6.0`
+
+The lieutenant-api container image to deploy.
+
+
+== `api.common_labels`
+
+[horizontal]
+type:: dict
+default: ::
+[source,yaml]
+----
+app.kubernetes.io/name: 'lieutenant-api'
+app.kubernetes.io/part-of: 'project-syn'
+----
+
+Common label to add to each resource of the lieutenant-api. Will be used
+as a label selector for pods.
+
+
+
+== `api.default_githost`
+
+[horizontal]
+type:: string
+default:: ``
+
+
+The githost to be used by default for new tenants.
+
+== `api.steward_image`
+
+[horizontal]
+type:: string
+default:: `docker.io/projectsyn/steward:v0.3.1`
+
+Image to use in generated `Steward` deployment manifests.
+
+
+
+== `api.ingress.host`
+
+[horizontal]
+type:: string
+default:: `lieutenant.todo`
+
+Defines the FQDN of the API ingress, should be overwritten on the cluster level.
+
+
+== `api.ingress.annotations`
+
+[horizontal]
+type:: dict
+default:: {}
+
+The annotations added to the created ingress. Needs to be set according to the
+deployed ingress controller.
+
+
+[source,yaml]
+----
+annotations:
+  cert-manager.io/cluster-issuer: letsencrypt-production
+  kubernetes.io/ingress.class: nginx
+  nginx.ingress.kubernetes.io/cors-allow-credentials: 'true'
+  nginx.ingress.kubernetes.io/cors-allow-methods: GET, POST, DELETE
+  nginx.ingress.kubernetes.io/cors-allow-origin: http://localhost:8080
+  nginx.ingress.kubernetes.io/enable-cors: 'true'
+----
+
+== `api.ingress.users`
+
+[horizontal]
+type:: list
+default: ::
+
+[source,yaml]
+----
+users:
+  - kind: ServiceAccount
+    name: api-access-vshn-portal-prod
+----
+
+A list of users that have permission to access the API. These entries translate
+to kubernetes subjects and can reference a `Group`, `User`, or `ServiceAccount`.
+Each specified `ServiceAccount` will be created.
+
+
+== `api.tenant_rbac`
+
+[horizontal]
+type:: list
+default:: []
+
+Role based access control to the created tenant resources. Lieutenant creates a `Role`
+for each tenant.
+
+
+[source,yaml]
+----
+tenant_rbac:
+  - tenant: 't-foo-324'
+    subjects:
+      - name: 'sa-bar'
+        kind: 'ServiceAccount'
+      - name: 'u-bar-1'
+        kind: 'User'
+  - tenant: 't-foo-1'
+    subjects:
+      - name: 'g-buzz'
+        kind: 'Group'
+      - name: 'u-bar-1'
+        kind: 'User'
+----
+
+The example configuration above will grant user `u-bar-1` and service account `sa-bar` read access to
+all Clusters owned by Tenant `t-foo-324`. And it will grant group `g-buzz` and user `u-bar-1`
+read access to all Clusters owned by Tenant `t-foo-1`.
+
+This can usually only be configured after the initial setup of lieutenant.
```

Comment:
```suggestion
This can usually only be configured after the initial setup of Lieutenant.
```

Please make sure you capitalize Lieutenant consistently in the docs.